docker/Dockerfile

```diff
@@ -26,26 +26,29 @@ RUN apt-get update \
   libffi-dev \
   vim \
  && apt-get clean
+RUN ln -sf /usr/bin/python${PYTHON_VERS} /usr/bin/python3
```

Comment:
```suggestion
RUN ln -sf /usr/bin/python${PYTHON_VERS} /usr/bin/python
```

Reply:
Why ? I find python misleading, some platform have python and python3 that does not point to the same version.
Furthermore all the python command in the nox targets and functional tests are called with `python3`.

Reply:
In the past, I has some issues with this. I can't remembe well but a program wanted `python` instead of `python3`. But anyway, as the ci uses already `python3` just ignore this comment.